tests/BasicSearchTests.FunctionalTests.Core/V3SearchFunctionalTests.cs

```diff
@@ -34,8 +34,26 @@ public class V3SearchFunctionalTests : BaseFunctionalTests
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-            Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
+            Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find at least some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task ResultForJsonNetHasDownloads()
```

Comment:
To match the rest of the file this should probably be `ShouldReturnResultsWithDownloads`